src/pages/briefs/[id]/applications/[applicationId].tsx

```diff
@@ -63,60 +59,25 @@ const ApplicationPreview = (): JSX.Element => {
 	const [success, setSuccess] = useState<boolean>(false)
 	const [openAccountChoice, setOpenAccountChoice] = useState<boolean>(false);
 
-	const accountSelected = async (account: WalletAccount): Promise<any> => {
-		try {
-			const result = await getAccountAndSign(account);
-			const resp = await authorise(
-				result?.signature as SignerResult,
-				result?.challenge!,
-				account
-			);
-			if (resp.status === 200 || resp.status === 201) {
-				setBalance(await getBalance(account.address, application.currency_id))
-			}
-			else {
-				setError({ message: "Could not connect wallet. Please Try again" })
-			}
-		} catch (error) {
-			setError(error)
-			console.log(error);
-		}
-	};
-
 	const router = useRouter();
 	const { id: briefId, applicationId }: any = router.query;
 
-	const getBalance = async (walletAddress: string, currency_id: number) => {
-		try {
-			const imbueApi = await initImbueAPIInfo();
-			const chainService = new ChainService(imbueApi, user);
-
-			if (!walletAddress) return "No Wallet Found"
-			const balance: any = await chainService.getBalance(walletAddress, currency_id)
-			return balance
-
-		} catch (error) {
-			setError(error)
-			console.log(error);
-		}
-	}
-
 	useEffect(() => {
 		const getSetUpData = async () => {
-			setLoading(true)
 			try {
+				const brief: Brief | undefined = await getBrief(briefId);
 				const applicationResponse = await fetchProject(applicationId);
+
 				const freelancerUser = await fetchUser(Number(applicationResponse?.user_id));
 				const freelancerResponse = await getFreelancerProfile(freelancerUser?.username);
-				const brief: Brief | undefined = await getBrief(briefId);
 				const userResponse = await getCurrentUser();
-				const balance = await getBalance(userResponse.web3_address, applicationResponse?.currency_id ?? 0)
 
-				setBalance(balance)
-				setFreelancer(freelancerResponse);
 				setBrief(brief);
 				setApplication(applicationResponse);
+				setBalance(balance)
```

Comment:
a bug here?
`balance` is not fetched or updated

Reply:
opps, forgot to remove balance states from application